utam-compiler/src/main/java/utam/compiler/helpers/MethodContext.java

```diff
@@ -68,7 +86,7 @@ public TypeProvider getReturnType(List<ComposeMethodStatement> statements,
         && methodReturnType != null
         && !lastStatementReturns.isSameType(methodReturnType)
         // Exclude load() method from this check, since it is always VOID type
-        && !methodName.equals(BEFORELOAD_METHOD_MANE)) {
+        && !methodName.equals("load")) {
```

Comment:
Nitpick: We're removing the constant because... ?

Reply:
because it's used in one place only. but i don't have strong opinion abt it, can reverse

Reply:
That's fair. We can leave it for now.